paimon-core/pom.xml

```diff
@@ -71,6 +73,12 @@ under the License.
             <version>${okhttp.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.squareup.okio</groupId>
+            <artifactId>okio-jvm</artifactId>
+            <version>${okio.version}</version>
+        </dependency>
+
```

Comment:
```
        <dependency>
            <groupId>com.squareup.okio</groupId>
            <artifactId>okio</artifactId>
            <version>${okio.version}</version>
        </dependency>
```
just add this is ok.

Reply:
ok, I'll try it first.

Reply:
Still : java.lang.ClassNotFoundException: kotlin.jvm.internal.Intrinsics

Now. I'll try shade okio.

> ```
>         <dependency>
>             <groupId>com.squareup.okio</groupId>
>             <artifactId>okio</artifactId>
>             <version>${okio.version}</version>
>         </dependency>
> ```
> 
> just add this is ok.

Reply:
I mean add okio, but still need to shade it. Because there is other version okio.